Lib/multiprocessing/shared_memory.py

```diff
@@ -68,6 +68,7 @@ class SharedMemory:
     _buf = None
     _flags = os.O_RDWR
     _mode = 0o600
+    _prepend_leading_slash = True if _USE_POSIX else False
```

Comment:
By defining a class attribute, we can permit this to be turned off easily in a simple subclass.
```
class NoSlashesSharedMemory(SharedMemory):
    _prepend_leading_slash = False
```

Agreed that it could have simply been set equal to ``_USE_POSIX`` though I thought this would better emphasize that it is a ``True`` for POSIX and ``False`` otherwise.  Perhaps this emphasis is unnecessary.

Reply:
I agree with you (see my comment later).